build.gradle

```diff
@@ -601,7 +601,7 @@ project(":logstash-integration-tests") {
         systemProperty 'org.logstash.integration.specs', rubyIntegrationSpecs
         environment "FEATURE_FLAG", System.getenv('FEATURE_FLAG')
         workingDir integrationTestPwd
-        dependsOn installIntegrationTestGems
+        dependsOn = [installIntegrationTestGems, copyProductionLog4jConfiguration]
```

Comment:
Nit/Question: Is this change necessary given that the `integrationTest` task already defines a `copyProductionLog4jConfiguration` dependency?

Reply:
Because running `./gradlew clean runIntegrationTests -PrubyIntegrationSpecs=specs/pipeline_log_spec.rb` the dependency on task `copyProductionLog4jConfiguration` seems to get lost and the copying of `log4j2.properties` file to `fixtures/logs_rollover/log4j2.properties` doesn't happen, resulting in:
```
Failures:

      1) Test Logstash Pipeline id rollover of pipeline log file when pipeline.separate_logs is enabled shouldn't create spurious file 
         Failure/Error: log_definition = File.read('fixtures/logs_rollover/log4j2.properties')
         
         Errno::ENOENT:
           No such file or directory - fixtures/logs_rollover/log4j2.properties
         # ./specs/pipeline_log_spec.rb:157:in `block in <main>'
         # /home/andrea/workspace/logstash_andsel/build/qa/integration/vendor/jruby/2.5.0/gems/logstash-devutils-2.2.1-java/lib/logstash/devutils/rspec/spec_helper.rb:51:in `block in <main>'
         # ./rspec.rb:32:in `<main>'

```